layouts/error.vue

```diff
@@ -2,15 +2,15 @@
     <div class="container">
         <Navigation />
         <Modal :isOpen="!hideCatsModal" :hideSubHeader="true" @close="hideCatsModal = true">
-            <template #header>Meow purrrr lick meow! 😺</template>
+            <template #header>Meow?</template>
             <template>
                 <div id="random-cat">
                     <span>{{ selectedQuote }}</span>
                     <img :src="selectedImage" alt="Picture of random cat." />
                 </div>
             </template>
             <template #actions>
-                <button @click="hideCatsModal = true" class="btn">Paw-don me</button>
+                <button @click="hideCatsModal = true" class="btn">Paw-down me</button>
```

Comment:
Why "paw-down me"? It was meant to be a cat pun with "pardon me".

Reply:
I completely understood the pardon me pun.
However, paw-down me still sounds like pardon me and can also be used to mean: put down me (put me down).
I'm also fine with keeping it as paw-don me.

Reply:
@AnAlgorithm Okay, that's fair! With the commit 27cc71a3a538d362c83fa0437c3f68fcc6127713, there is a 50-50 chance of either message appearing as the text in the close button.

Reply:
cool!